Its.Log.UnitTests/TelemetryTests.cs

```diff
@@ -107,5 +107,18 @@ public void Telemetry_events_contain_an_operation_name()
                            .Be("hello!");
         }
 
+        [Test]
+        public async Task The_user_identifier_can_be_specified_programmatically()
+        {
+            using (Log.With<Telemetry>(t => t.UserIdentifier = "epic_user").Enter(() => { }))

```

Comment:
How do you handle when UserIdentifier is not set?  For example, Telemetry on a Login page where nobody is currently a "user".